example/styleguide.json

```diff
@@ -1,5 +1,9 @@
 {
   "title": "React Style Guide",
+  "typekit": {
+    "id": "rtq7sdx",
```

Comment:
This code is accessible in every page. You just have to inspect source. I don't believe it's a problem. Do you think we should remove this?